src/modules/builder/nav-bar/components/TemplatesSlider.tsx

```diff
@@ -21,7 +21,7 @@ export const TemplateSlider = () => {
       splideInstanceRef.current = new SplideCore(targetElement, {
         perPage: 2,
         pagination: false,
-        gap: '33px',
+        gap: '-72px',
```

Comment:
why negative?

Reply:
I increased the container width, so the gap between items in the slider list looked huge with any positive values. Not sure how to adjust it, without using negative values.

Reply:
made changes in the gap part